CMSIS/DoxyGen/NN/src/history.txt

```diff
@@ -9,10 +9,10 @@
     <tr>
     <td>V1.x.0</td>
     <td>
-      Added wrapper functions for convolution related APIs
       <ul>
         <li>arm_depthwise_conv_wrapper_s8</li>
         <li>arm_convolve_wrapper_s8</li>
+        <li>arm_nn_sat_doubling_high_mult_no_sat</li>
```

Comment:
I'm just wondering if the name could be confusing containing both "sat" and "no_sat". Suggest just "arm_nn_doubling_high_mult()" or "arm_nn_doubling_high_mult_no_sat()"

Reply:
Thanks for the comment. I have gone with arm_nn_doubling_high_mult() for the saturating version and arm_nn_doubling_high_mult_no_sat() for the non saturating version. And I changed the names for the mve versions as well so it is uniform.